### DISCUSSION TITLE
vercel certs issue "Error! ACME request was malformed and responded with an error"

### DISCUSSION THREAD

**Discussion by adnaz** [vercel/community#282] [Help] [answered]
I added the TXT record in my registrar to be able to  solve the DNS challenge (vercel certs issue  blog.ispapp.co --challenge-only)
but when i run  "vercel certs issue blog.ispapp.co" i get this error  "Error! ACME request was malformed and responded with an error"
and when i run it for the 2nd time i get this "Error! We could not solve the dns-01 challenge for cns blog.ispapp.co".

Any help would be appreciated 🍻

**Comment by Cl3arglass** [comment, marked as answer]
Hello, thank you for reaching out. If you run the command `dig TXT _acme-challenge.blog.ispapp.co +short` does the TXT record you were expecting get returned?

**Reply by adnaz** [reply]
yeah it returned the value  generated using --challenge-only

**Reply by Cl3arglass** [reply]
Hello, checking in it appears that a cert for "blog.ispapp.co" has not been generated yet. Would you be able to run `vercel certs issue blog.ispapp.co` again to see what is returned?

**Reply by Cl3arglass** [reply]
Looking further, it appears that there is an issue with your domain's nameservers in that ns2.goodnotion.com is not responding:
`dig A @ns2.goodnotion.com. blog.ispapp.co

; <<>> DiG 9.10.6 <<>> A @ns2.goodnotion.com. blog.ispapp.co
; (1 server found)
;; global options: +cmd
;; connection timed out; no servers could be reached`

With that we can recommend reaching out to your dns provider for more insight or switching to Vercel's nameswervers.

**Reply by crowterli** [reply]
Just for completions sake, ns1.goodnotion.com is fine:

```
% dig blog.ispapp.co @ns1.goodnotion.com +short
76.76.21.21
```

I'm going to mark this as an answer but if there are still issues just let us know.
